modules/dxp/apps/portal-workflow/portal-workflow-metrics-web/test/components/App.es.js

```diff
@@ -139,6 +139,7 @@ describe('The App component should', () => {
 	});
 
 	test('Render the process metrics page on performance tab and back to dashboard', () => {
+		const metricsCalculated = findByText('Metrics calculated');
```

Comment:
Hey @ramonmsp, Is the `findByText` case insensitive? Thx

Reply:
It was wrong, sorry :(